engine/src/main/java/org/terasology/world/generation/BaseFacetedWorldGenerator.java

```diff
@@ -83,6 +89,10 @@ public World getWorld() {
         return world;
     }
 
+    /**
+     * Returns current {@link WorldBuilder} or creates new if none exists
```

Comment:
The way your sentence is structured, it will
- Return existing one if it exists
- Or create a new one (and not return anything).
Rephrase it. The job of a javadocs writer is not just to convert `getWorldBuilder` to `Returns world builder`.

For reference, this is how I would personally write it:
`Returns the current WorldBuilder used by this WorldGenerator. If no WorldBuilder has been initialized so far, a new one is created and then returned.`
Kinda verbose, but much more clear in meaning.

Reply:
Also, don't leave the `@return` blank.
`@return The WorldBuilder used by this WorldGenerator` or something.
I realize there is some duplication, but it's inevitable.